### PR TITLE
Drawer overhaul

### LINKED ISSUE #568: implement new drawer design
toggl-android-nav-more

### PR DESCRIPTION
**Pull request by taavirehemagi**
Lifted drawer above the action bar, which is a bit of a hack and affects layouts across the app. Still, the drawer is below the system status bar, unlike in the design guideline, but it's consistent with other apps following material design. Not sure, if lifting it over the status bar is needed or even possible. Also, sync bar redesign probably needed in future.

Closes #568

### CODE REVIEW

File: Joey/UI/Activities/MainDrawerActivity.cs [hunk 1]
```diff
@@ -107,6 +132,16 @@ protected override void OnCreateActivity (Bundle bundle)
             }
         }
 
+        private void OnUserChangedEvent (object sender, System.ComponentModel.PropertyChangedEventArgs args)
```
Comment (masterrr) [inline Joey/UI/Activities/MainDrawerActivity.cs]: What do you think on importing the System.ComponentModel namespace to improve readability? (in the usings on the top)

Comment (taavirehemagi) [inline Joey/UI/Activities/MainDrawerActivity.cs]: Ah, of course :+1:

**Comment by masterrr** [comment]
I say yes :+1: :smile: 
Though there are merging conflicts to be fixed

**Comment by monday8am** [comment]
- Not sure, if lifting it over the status bar is needed or even possible
below is ok in my opinion. 
- More descriptive filename than Decor : Joey/Resources/layout/Decor.axml
- Needs to be rebase before merge.
- Check the code with the option Xamarin > Preferences > Text Editor > Source Analysis . You'll see code that you can improve and unused imports.

**Comment by taavirehemagi** [comment]
Rebased from android-redesign branch. The drawer will be over the action bar once the toolbar is implemented from another PR.

**Pull request merged**